### ISSUE TITLE
Notification icon showing notifications even after reading the notifications 

### ISSUE THREAD

**Issue by FaiqaShafqat79** [LibreHealthIO/lh-ehr#1414]
**A)	My Outreachy username :** faiqa 

**B)	Issue Title :** Notification icon showing notifications even after reading the notifications 

**C)	Bug Report Date :** March 5, 2019 

**D)	Site Affected :** Documentation Site. 

**E)	OS/Browser Used :** Windows/Chrome. 

**F)	Workflow Module/Tab :** Message and Reminder Center. 

**G)	Step to reproduce :** 

 1. Login with your account credentials. 
 2. Take your cursor to the notifications icon in the top right corner. 
 3. Click on the icon. 
 4. Message and Reminder Center tab will open. 

**H)	Expected Behavior :** When we click on notification icon it should display the notified message/notification and the numbering on the icon should gone. 

**I) What actually Happened :** When I clicked on notification icon it showed the notified message but numbering on icon remains the same. 

**J) ScreenShot :** 
 
![notification 1](https://user-images.githubusercontent.com/37265494/53776558-3277fc80-3f18-11e9-8dd8-ae613e807306.PNG)

**K) Estimation of the bug severity :** Module function continues with ongoing errors

**Comment by muarachmann** [comment]
@FaiqaShafqat79 triggering a change of status in the database once this read will do the job. I think for now if you change from read to done and reload the page you shouldn't see the notification count increment again. so that is a quick walk around - i.e turning it to the done function once the message is opened

**Comment by FaiqaShafqat79** [comment]
Yeah it's a quick walk around if we trigger a change of status in database

**Comment by aethelwulffe** [comment]
It is a nice tweak to do.  Better yet, find two places it can be used, and write a standard library for page reload arguments.
